wp-content/themes/csisnuclearnetwork/template-parts/post-related-content.php

```diff
@@ -18,24 +18,40 @@
 	// vars
 	$related_posts = get_field( 'related_posts', $term );
 
-	// if related content we show them
-	if ( $related_posts || has_tag() ) {
-		echo '<h2 class="single__footer-heading">';
-			echo nuclearnetwork_get_svg( "3-arrows" );
-			_e( 'Related Content', 'nuclearnetwork' );
-		echo '</h2>';
-		nuclearnetwork_display_tags();
-
-		if ( $related_posts ) {
-			echo '<ul class="related-posts" role="list">';
-				foreach( $related_posts as $post ):
-					setup_postdata($post);
-					echo '<li>';
-					get_template_part( 'template-parts/block-post-related' );
-					echo '</li>';
-				endforeach;
-			echo '</ul>';
-			wp_reset_postdata();
-		}
-	}
+	// if no related content we bail
+	if ( !($related_posts || has_tag() || has_category()) ) {
+      return;
+    }
+
+    echo '<hr />';
+    echo '<h2 class="single__footer-heading">';
```

Comment:
Since this heading doesn't match the rest of the headings in the footer, you'll need to use a different class.